command/sub8_missions/missions/buoy.py

```diff
@@ -55,3 +42,17 @@ def run(sub):
     # yield sub.move.forward(
 
     print "Bumped the buoy"
+
+@util.cancellableInlineCallbacks
+def get_buoy_tf(sub):
```

Comment:
Reason for bringing out into its own function: I'm gonna use this a few times in the buoy mission to make sure we have the right target as we get closer